ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -780,7 +782,9 @@ export default defineComponent({
     isMhrTransfer: { type: Boolean, default: false },
     hideRemovedOwners: { type: Boolean, default: false },
     showChips: { type: Boolean, default: false },
-    validateTransfer: { type: Boolean, default: false }
+    validateTransfer: { type: Boolean, default: false },
+    hideTableErrors: { type: Boolean, default: false },
```

Comment:
@cameron-eyds added these additional props to:
1. Prevent errors showing up (they were showing up in Prev Owners if actual Home Owners had errors)
2. Force show groups for TIC tenancy types

Reply:
Sounds good!